x-pack/plugins/apm/public/components/shared/charts/transaction_charts/ml_header.tsx

```diff
@@ -62,7 +62,7 @@ export function MLHeader({ hasValidMlLicense, mlJobId }: Props) {
         'xpack.apm.metrics.transactionChart.machineLearningTooltip',
         {
           defaultMessage:
-            'The stream around the average duration shows the expected bounds. An annotation is shown for anomaly scores ≥ 75.',
+            'The shaded area shows the expected latency. An red annotation is shown for anomaly scores ≥ 75.',
```

Comment:
```suggestion
            'The stream around the average duration shows the expected bounds. A red vertical annotation indicates anomalies with an anomaly score of 75 or above.',
```

Reply:
Did you intentionally add "average duration" back?

Reply:
> Did you intentionally add "average duration" back?

I might have 🙈 I've fixed the copy now